projects/ngrx.io/content/guide/store/action-groups.md

```diff
@@ -0,0 +1,67 @@
+# Action Groups
+
+Action groups is a feature that allows you to group actions together. While writing actions, the actions in most of the cases looks like below.
+
+<code-example header="products-page.actions.ts">
+import { createAction, props } from '@ngrx/store';
+// defining an action without payload
+export const opened = createAction('[Products Page] Opened');
+// defining an action with payload using the `props` function
+export const paginationChanged = createAction(
+  '[Products Page] Pagination Changed',
+  props&lt;{ page: number; offset: number }&gt;()
+);
+// defining an action with payload using the props factory
+export const queryChanged = createAction(
+  '[Products Page] Query Changed',
+  (query: string) => ({ query })
+);
+</code-example>
+
+
+With the help of this new `createActionGroup` api, we can avoid the above code and write the following code.
```

Comment:
The docs shouldn't mention if it's new or old.

```suggestion
With the help of the `createActionGroup` API, we can avoid the above code and write the following code.
```

Reply:
Should we mention here why/when `createActionGroup` should be used?
E.g. to group actions that have the same source, making the code more compact, etc, etc